tutorial_src/gue/config.yml

```diff
@@ -0,0 +1,7 @@
+title: Guess the number
+
+extra_compilation_files: ['guelib.h', 'guelib.cpp', 'guelib.i']
+extra_compilation_args: {'cpp': 'guelib.cpp'}
+
+memory_limit: 65536
```

Comment:
Pamiętajmy że limit 64MB ma być limitem dostępnym dla zawodnika, tutaj kompletnie nie jest uwzględniona górka związana z narzutem biblioteczki.

Reply:
Jest to na pewno słuszna rzecz do napisania w tutorialu.
Ale jaka jest tutaj górka? Zauważmy że zużywam stałą (i bardzo małą) pamięć.
Mam dorzucić eps (1kb) do pamięci?
Czy są narzuty związane z samym istnieniem biblioteczki?

Reply:
Nie umiem precyzyjnie odpowiedzieć na to pytanie.
Pamiętaj że include to nie jest tylko narzut tego ile zadeklarowałeś w biblioteczce.
To też jakieś rzeczy związane z tym jak kompilator to przemielił.
Może size -A na obiekcie skompilowanej biblioteczki daje jakieś wybierne informacje na ten temat.
Może @accek wiesz jak powinno się dobrze szacować takie rzeczy?

Reply:
Zwiększone na czuje